src/Transfer/Sources/NHost.php

```diff
@@ -228,6 +242,42 @@ public function convertAttribute(array $column): Attribute
         }
     }
 
+    /**
+     * Convert Index
+     * 
+     * @param string $table
+     * @return Index|false
+     */
+    public function convertIndex(array $index): Index|false
+    {
+        $pattern = "/CREATE (?<unique>UNIQUE)? INDEX (?<name>\w+) ON (?<table>\w+\.\w+) USING (?<method>\w+) \((?<columns>\w+)\)/";
```

Comment:
> `\((?<columns>\w+)\)/`

Columns will be comma-separated and can include the order afterward e.g. `(title ASC, name, year DESC)`. ASC is the default if no order is specified. Those orders should be passed to the `Index` constructor